docs/prow/dependency-update.md

```diff
@@ -0,0 +1,68 @@
+# Dependency updates
+
+To help keep dependency up to date you can use dependabot together with automated testing and merging of pull requests.
+
+Dependabot is a GitHub tool which can monitor repository and create pull requests to update dependencies. Dependabot support multiple languages. Check dependabot documentation for details.
+
+Pull requests created by dependabot will be tested by Prow. Pull requestes with required tests passed and meeting criteria can be automatically merged by Tide without human review.
+
+Owners of updated files can subscribe to receive notifications when pull request is merged without human review.
+
+## How to use it
+
+Enable dependabot in your repository. [Here](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates) is detailed dependabot documentation.
+
+To enable dependabot create dependabot.yml configuration file. To make it easier, you can use a rendertemplate tool to create dependabot.yml file. Use [kyma-project/test-infra](../../.github/dependabot.yml) configuration as an example.
```

Comment:
```suggestion
2. To enable Dependabot, create `dependabot.yml` configuration file. You can use a rendertemplate tool to create the `dependabot.yml` file. Use [kyma-project/test-infra](../../.github/dependabot.yml) configuration as an example.
```
Wait, didn't the previous step also say "create Dependabot"? Are we doing it again now? What was the purpose of the previous step?

Reply:
Which previous step? A sentence before says that user need to enable dependabot. There is nothing about creating a file.